wiki/Installing-Oppia-Android.md

```diff
@@ -526,23 +481,33 @@ bazel test //domain/src/test/java/org/oppia/android/domain/onboarding:AppStartup
 
 - To learn about how we write tests at oppia-android, please refer to: [Oppia Android Testing](https://github.com/oppia/oppia-android/wiki/Oppia-Android-Testing).
 
-- To learn more about getting started with Bazel, please see [Oppia-Bazel-Glossary](https://github.com/oppia/oppia-android/wiki/Oppia-Bazel-Glossary).
+- To learn more about getting started with Bazel, please see the [Bazel User Guide](https://github.com/oppia/oppia-android/wiki/Bazel-User-Guide).
```

Comment:
```suggestion
- An important reference to go back to as you continue developing with Bazel is the [Bazel User Guide](https://github.com/oppia/oppia-android/wiki/Bazel-User-Guide).
```

'Getting started' sounds a bit too much like 'start here' which isn't correct--_this_ page is the actual starting place and the guide is just a long-term reference.

Reply:
Done.